### PR TITLE
PLAT-142964: Sampler: Fix global knob setting resets when component changes

### PR DESCRIPTION
**Pull request by YunhoLee0**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
The global knob setting stays even if the story changes.

### Resolution
Referred PR: https://github.com/enactjs/sandstone/pull/906

### Links
PLAT-142964

### CODE REVIEW

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js [hunk 1]
```diff
@@ -211,21 +213,27 @@ const StorybookDecorator = (story, config) => {
...
-	const skinKnobs = {};
-	if (!allSkins) {
-		skinKnobs.skin = select('skin', skins, Config, currentSkin);
+
+	globals.locale = select('locale', locales, Config, localeFromURL || globals.locale);
+	globals.allSkins = boolean('show all skins', Config, globals.allSkins);
+	globals.nightMode = boolean('night mode', Config, globals.nightMode);
+
+	let skinKnobs = {};
+	let {accent, highlight} = {};
```
Comment (MikyungKim) [inline samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js:227]:
```suggestion
	let accent, highlight;
```

**Review by MikyungKim — CHANGES_REQUESTED** [review]
I couldn't find selectors for `accent` and `highlight` in global knobs. Would you take a look?
image

**Comment by YunhoLee0** [comment]
If you click the knobs, the color selector would be displayed.
image

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js (hunk 1 above)
Comment (YunhoLee0) [reply samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js]: OK

**Comment by YunhoLee0** [comment]
The 'color' function of the storybook's knob has a bug.
It doesn't return a new value when the preferred value is changed. 
`globals.accent = color('accent', preferredValue, groupId);`
So I also added a workaround to handle it.

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js [hunk 2]
```diff
@@ -224,15 +226,32 @@ const StorybookDecorator = (story, config) => {
...
+
+	if (globals.defaultSkinStyles || isSkinChanged) {
+		globals.accent = defaultColors[globals.skin].accent;
+		globals.highlight = defaultColors[globals.skin].highlight;
+	}
+
+	if (!globals.defaultSkinStyles) {
+		// NOTE: 'color' function has a bug. It doesn't return a changed value when the value (2nd parameter) is changed. It seems it doesn't reload new value.
+		globals.accent = color('accent', accentFromURL || globals.accent || Config.defaultProps.accent, Config.groupId);
+		globals.highlight = color('highlight', highlightFromURL || globals.highlight || Config.defaultProps.accent, Config.groupId);
```
Comment (MikyungKim) [inline samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js:248]: Config.defaultProps.accent -> Config.defaultProps.highlight ?

**Comment by YunhoLee0** [comment]
Since the 'color' function's bug, I reverted it.

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js (hunk 2 above)
Comment (YunhoLee0) [reply samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js]: The code does not exist cause reverted.
Thank you for the review :)

**Review by MikyungKim — APPROVED** [review]
:shipit:

**Pull request merged**